core/templates/dev/head/header_css_libs.html

```diff
@@ -16,3 +16,4 @@
 <link rel="stylesheet" type="text/css" media="screen"
       href="/css/oppia.css">
 
+
```

Comment:
Let's not edit this file at all (to make the commit cleaner). (It shows up as a changed file at https://github.com/oppia/oppia/pull/1732/files)